lisa/tools/ntttcp.py

```diff
@@ -403,6 +405,18 @@ def create_ntttcp_tcp_performance_message(
         other_fields["pkts_interrupts"] = client_result.pkts_interrupt
         other_fields["sender_cycles_per_byte"] = client_result.cycles_per_byte
         other_fields["receiver_cycles_rer_byte"] = server_result.cycles_per_byte
```

Comment:
need add other_fields["protocol_type"] = TransportProtocol.Tcp in create_ntttcp_tcp_performance_message